docusaurus/docs/iOS/06-advanced/04-screensharing.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Screen sharing
+description: Setup for screen sharing
+---
+
+## Introduction
+
+The StreamVideo iOS SDK has support for displaying screensharing tracks, as well as screensharing from an iOS device. There are two options for screensharing from an iOS device:
+- in-app screensharing - the screen is shared only while the app is active.
+- broadcasting - the device shares the screen even when the app goes into the background.
+
+Both of these options use Apple's framework `ReplayKit` for broadcasting the user's screen. 
+
+In order for a user to be able to share their screen, they must have the `screenshare` capability configured for the call they are in.
+
+## In-app screensharing
+
+In-app screensharing broadcasts only the app's screens. When you are in a call, and you have the required capability, you can start screensharing by calling the `startScreensharing` method, with the `inApp` screensharing type:
+
+```swift
+Task {
+    let call = streamVideo.call(callType: "default", callId: "123")
+    try await call.join()
+    try await call.startScreensharing(type: .inApp)
+}
+```  
+
+If you use our UI components and the `CallViewModel`, the same method is also available from there.
+
+When the method is invoked, `ReplayKit` will ask for the user's consent that their screen will be shared. Only after the permission is granted, the screensharing starts.
```

Comment:
Do we have handling for the case where user denies to grant permission?

Reply:
that's handled by Apple's component. We won't receive the start broadcast notification, and it just stays in the `notStarted` state.